pkg/kubelet/cm/node_container_manager_linux.go

```diff
@@ -140,12 +140,15 @@ func enforceExistingCgroup(cgroupManager CgroupManager, cName CgroupName, rl v1.
 	// Enforce MemoryQoS for cgroups of kube-reserved/system-reserved. For more information,
 	// see https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2570-memory-qos
 	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.MemoryQoS) {
+		if rp.Unified == nil {
+			rp.Unified = make(map[string]string)
```

Comment:
Does unified imply only cgroupsv2? Do we have a comment/docs along those lines somewhere?

Reply:
In containerd, there is logic like https://github.com/containerd/containerd/blob/main/pkg/cri/opts/spec_linux.go#L541-L551. 

And for `./vendor/k8s.io/cri-api/pkg/apis/runtime/v1/api.pb.go`
```
	// Unified resources for cgroup v2. Default: nil (not specified).
	// Each key/value in the map refers to the cgroup v2.
	// e.g. "memory.max": "6937202688" or "io.weight": "default 100".
	Unified map[string]string `protobuf:"bytes,9,rep,name=unified,proto3" json:"unified,omitempty" protobuf_key:"bytes,1,opt,name=key,proto3" protobuf_val:"bytes,2,opt,name=value,proto3"`
```